README.md

```diff
@@ -2,31 +2,33 @@
 
 # Summer-Ray-Tracer
 
-SJTU ACM Honor Class 2023 Ray Tracer Project With Rust !
+> SJTU ACM Honors Class 2023 Ray Tracer Project with Rust!
 
-**Before you start, please read this README carefully**, there is no useless information in this document.
+**Before you start, please read this README carefully**, and there is no useless information in this document.
 
 ### Introduction
 
-Briefly, you need to build a toy ray tracer in this project, with the Rust language.
+Briefly, you need to build a toy ray tracer in this project with the Rust language.
 
-### Where can you learn about ray tracing?
+### Where can you Learn about Ray Tracing?
 
-Out project is based on the [Ray Tracing In One Weekend](https://raytracing.github.io/) book series, containing three books, which is great resource for beginners to learn about ray tracing.
+Our project is based on the [***Ray Tracing In One Weekend***](https://raytracing.github.io/) book series, containing three books, which is a great resource for beginners to learn about ray tracing.
```

Comment:
Unnecessary formatting. Avoid **bold** and *italic* text, especially ***bold and italic*** text.